contracts/src/core/EigenDAThresholdRegistry.sol

```diff
@@ -0,0 +1,105 @@
+        quorumAdversaryThresholdPercentages = _quorumAdversaryThresholdPercentages;
+    }
+
+    function updateQuorumConfirmationThresholdPercentages(bytes memory _quorumConfirmationThresholdPercentages) external onlyOwner {
+        quorumConfirmationThresholdPercentages = _quorumConfirmationThresholdPercentages;
+    }
+
+    function updateQuorumNumbersRequired(bytes memory _quorumNumbersRequired) external onlyOwner {
+        quorumNumbersRequired = _quorumNumbersRequired;
+    }
+
+    function updateDefaultSecurityThresholdsV2(SecurityThresholds memory _defaultSecurityThresholdsV2) external onlyOwner {
+        defaultSecurityThresholdsV2 = _defaultSecurityThresholdsV2;
+    }
+
+    function addVersionedBlobParams(VersionedBlobParams memory _versionedBlobParams) external onlyOwner returns (uint16) {
+        return _addVersionedBlobParams(_versionedBlobParams);
+    }
+
+    function _addVersionedBlobParams(VersionedBlobParams memory _versionedBlobParams) internal returns (uint16) {
+        versionedBlobParams[nextBlobVersion] = _versionedBlobParams;
+        emit VersionedBlobParamsAdded(nextBlobVersion, _versionedBlobParams);
+        return nextBlobVersion++;
+    }
+
+    ///////////////////////// V1 ///////////////////////////////
+
+    /// @notice Gets the adversary threshold percentage for a quorum
+    function getQuorumAdversaryThresholdPercentage(
+        uint8 quorumNumber
+    ) public view virtual returns (uint8 adversaryThresholdPercentage) {
+        if(quorumAdversaryThresholdPercentages.length > quorumNumber){
+            adversaryThresholdPercentage = uint8(quorumAdversaryThresholdPercentages[quorumNumber]);
+        }
+    }
+
+    /// @notice Gets the confirmation threshold percentage for a quorum
+    function getQuorumConfirmationThresholdPercentage(
+        uint8 quorumNumber
+    ) public view virtual returns (uint8 confirmationThresholdPercentage) {
+        if(quorumConfirmationThresholdPercentages.length > quorumNumber){
+            confirmationThresholdPercentage = uint8(quorumConfirmationThresholdPercentages[quorumNumber]);
+        }
+    }
+
+    /// @notice Checks if a quorum is required
+    function getIsQuorumRequired(
+        uint8 quorumNumber
+    ) public view virtual returns (bool) {
+        uint256 quorumBitmap = BitmapUtils.setBit(0, quorumNumber);
+        return (quorumBitmap & BitmapUtils.orderedBytesArrayToBitmap(quorumNumbersRequired) == quorumBitmap);
+    }
+
+    ///////////////////////// V2 ///////////////////////////////
+
+    /// @notice Gets the default security thresholds for V2
+    function getDefaultSecurityThresholdsV2() external view returns (SecurityThresholds memory) {
+        return defaultSecurityThresholdsV2;
+    }
+
+    /// @notice Returns the blob params for a given blob version
+    function getBlobParams(uint16 version) external view returns (VersionedBlobParams memory) {
+        return versionedBlobParams[version];
```

Comment:
same here, should we check `version < nextBlobVersion`?

Reply:
dont think it is totally necessary to require on the getter